code/modules/mob/mob_movement.dm

```diff
@@ -17,8 +17,8 @@
 
 //The byond version of these verbs wait for the next tick before acting.
 //	instant verbs however can run mid tick or even during the time between ticks.
-#define DO_MOVE(this_dir) var/final_dir = turn(this_dir, -dir2angle(dir)); Move(get_step(mob, final_dir), final_dir);
-
+//#define DO_MOVE(this_dir) var/final_dir = turn(this_dir, -dir2angle(dir)); Move(get_step(mob, final_dir), final_dir);

```

Comment:
Are there any reason to keep these commented instead of deleted?

Reply:
Nope, this is a byproduct of testing so I could easily go back to the old system